### ISSUE TITLE
erro: no vpn configuration file found

### ISSUE THREAD

**Issue by sugaratias333** [wfg/docker-openvpn-client#115]
I am passing in CONFIG_FILE as documented, but the container just outputs 'err: no vpn configuration file found',  no matter what I do.

**Comment by Pierric82** [comment]
Same here. Tried with CONFIG_FILE and without, including /config/ and not including it, but no luck.

**Comment by Pierric82** [comment]
This is actually because the image currently available at ghcr.io/wfg/openvpn-client does not match the codebase in this repo! it seems outdated.
It will start if you change CONFIG_FILE=something.ovpn to VPN_CONFIG_FILE=something.ovpn , and the mount from /config to /data/vpn
I haven't had time yet to check if it works, though!
The alternative might be to build the image locally from this repo.

**Comment by sugaratias333** [comment]
Good spot.  I shall try this later on and verify it on mine also.

**Comment by sugaratias333** [comment]
Yes, I can confirm that does allow the container to startup. Thanks!

**Comment by MortenStabenau** [comment]
+1 for this issue

@wfg would it be possible to update the container in the registry?

**Comment by juev** [comment]
+1 the same problem
@wfg could you update the version in the registry?

**Comment by juev** [comment]
The master branch contains all the necessary changes to use the configuration files.
But the docker image must be rebuilt with the creation of a new tag in the repository.

There are also a number of problems in entry.sh file, corrected in PR #116 

@wfg
